plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -5400,7 +5400,7 @@ protected AttachIsoAnswer execute(AttachIsoCommand cmd) {
             String isoDatastorePath = String.format("[%s] %s%s", storeName, isoStorePathFromRoot, isoFileName);
 
             if (cmd.isAttach()) {
-                vmMo.attachIso(isoDatastorePath, morSecondaryDs, true, false, cmd.getDeviceKey());
+                vmMo.attachIso(isoDatastorePath, morSecondaryDs, true, false, cmd.getDeviceKey(), cmd.isForce());
```

Comment:
_cmd.isForced()_ here

Reply:
@sureshanaparti isForce() is a function of AttachIsoCommand(not AttachCommand), that already existed, and I'm just calling that.

Reply:
OK @Pearl1594